src/components/navigation-header/components/primary-nav-link/index.tsx

```diff
@@ -1,29 +1,23 @@
 import Link from 'next/link'
 import useCurrentPath from 'hooks/use-current-path'
-import { useCurrentProduct } from 'contexts'
 import Text from 'components/text'
 import s from './primary-nav-link.module.css'
 
 interface PrimaryNavLinkProps {
 	ariaLabel: string
 	navItem: {
-		id?: string
-		isSubmenu?: boolean
 		label: string
-		pathSuffix?: string
+		url: string
```

Comment:
We want to link to the Terraform Registry in the top nav (https://registry.terraform.io/). In order to achieve this, I've modified the interface and component slightly here to allow for full URLs, rather than strictly requiring a path suffix only.

The URL construction that previously happened inside this component now happens outside of it. Hopefully this aligns with our goal of separating logic and presentational concerns as well.